pilot/pkg/config/clusterregistry/clusterregistry.gotmpl

```diff
@@ -0,0 +1,16 @@
+apiVersion: clusterregistry.k8s.io/v1alpha1
+kind: {{.Kind}}
+metadata:
+  name: {{.Name}}
+  annotations:
+    config.istio.io/pilotEndpoint: "{{.PilotIP}}:9080"
+    config.istio.io/platform: "{{if .Platform}}{{.Platform}}{{else}}Kubernetes{{end}}"
+    {{if .PilotCfgStore}}config.istio.io/pilotCfgStore: "{{.PilotCfgStore}}"
+    {{end -}}
+    config.istio.io/accessConfigSecret: "{{.AccessConfigSecret}}"
+    config.istio.io/accessConfigSecretNamespace: "{{.AccessConfigNamespace}}"
```

Comment:
How about: default is same namespace as pilot ( I believe it is available in an env variable ). 

If advanced users need to override great, but no need to confuse users.

Reply:
I agree with @costinm you should check if the namespace pilot is running is is available already and then use it.   If there is a use case for having the secret in a different namespace that the one pilot is running in could you please describe it.